themes/stanford_basic/dist/js/behaviors.js

```diff
@@ -125,6 +125,15 @@ window.Drupal.behaviors.stanford_basic = {
         }
       }
 
+      /**
```

Comment:
@pookmish I added the new class here with JS. I think I could also do it in the theme file, right?

Reply:
Yes, this should not be done in JS.

Reply:
Rats. Ok.